news-bundle/src/Resources/contao/modules/ModuleNewsReader.php

```diff
@@ -16,13 +16,15 @@
 use Contao\CoreBundle\Routing\ResponseContext\HtmlHeadBag\HtmlHeadBag;
 
 /**
- * Front end module "news reader".
+ * Front end module "newsreader".
  *
  * @property Comments $Comments
  * @property string   $com_template
  * @property array    $news_archives
  *
  * @author Leo Feyer <https://github.com/leofeyer>
+ *
+ * @todo Rename to ModuleNewsreader in Contao 5.0
```

Comment:
Why? Then we also need to rename `ModuleNewsList` to `ModuleNewslist`, don't we?

Reply:
No, because "newslist" is not a word but "newsreader" is. 🤷‍♂️

Reply:
But that also seems debatable.

Reply:
It seems that "newsreader" means "Nachrichtensprecher", so we should actually stick with "news reader". 🤦‍♂️ I am going to adjust the code inspector accordingly.

Reply:
Reverted in f6a1068b74b819a2874aa9f74f25b4d2169b92ed.

Reply:
I see - yes that should be changed.